runtime/README.md

```diff
@@ -790,7 +799,7 @@ Command Code: `0x4154_5348` ("ATSH")
 | --------      | -------- | ---------------
 | chksum            | u32      | Checksum over other output arguments, computed by Caliptra. Little endian. |
 | fips_status      | u32      | Indicates if the command is FIPS approved or an error.                     |
-| auth_req_result | u32      | AUTHORIZE_IMAGE: 0xDEADC0DE and DENY_IMAGE_AUTHORIZATION: 0x21523F21    |
+| auth_req_result | u32      | AUTHORIZE_IMAGE (0xDEADC0DE), IMAGE_NOT_AUTHORIZED (0x21523F21) or IMAGE_HASH_MISMATCH (0x8BFB95CB) |
```

Comment:
For most other commands (like signature verify commands) for a negative results, we report a mailbox command failure and put the reason in the non-fatal error code reg. Just want to call that out to be sure we want this to behave differently.

On the other hand, DPE does report a result that can be a negative result with a "mailbox complete". But that's maybe a bit of an exception since it's a whole separate submodule.

Reply:
Ack.